docs/UserGuide.md

```diff
@@ -291,9 +291,10 @@ Adds a course you teach to the list of courses.
 * `add c/CS2103T`
 * `add c/CS2100`
 * `add c/GEA1000`
+* `add c/ST2334` returns the following result:
+  *![Add_Course](images/CommandSuccessScreenshots/AddCourseSuccess.png)*
```

Comment:
Good addition of screenshots to demonstrate how the app should like like at after executing different commands, which users should find useful